exo-add/src/cli.ls

```diff
@@ -60,6 +62,15 @@ function help
   console.log help-message
 
 
+function check-for-service(app-config, data)
```

Comment:
I'd also argue that the order of these parameters is not intuitive, and they should therefore be named parameters.